docs/user-guide/command-line-interface.md

```diff
@@ -440,6 +441,16 @@ Example:
 
     eslint "src/**/*.js" --cache --cache-location "/Users/user/.eslintcache/"
 
+#### `--cache-strategy`
+
+Strategy for the cache to use for detecting changed files. Can be either `metadata` or `content`. If no strategy is specified, `metadata` will be used.
+
+The `content` strategy can be useful in cases where the modification time of your files change even if their contents have not.
```

Comment:
I wonder if it would be helpful to mention the common reason why this happens, which I believe is this:

> 
```suggestion
The `content` strategy can be useful in cases where the modification time of your files change even if their contents have not. For example, this can happen during git operations like git clone because git does not track file modification time.
```

Reply:
I like it! :+1:

Reply:
Thanks 😃